spring-xsuaa-it/src/test/java/com/sap/cloud/security/xsuaa/MockXSUAAServerConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package com.sap.cloud.security.xsuaa;
```

Comment:
please rename the class name so that it's in camelCase and with that it'll be consistent across the lib